client/app/bundles/course/assessment/submission/components/WarningDialog.tsx

```diff
@@ -46,7 +52,7 @@ const WarningDialog: FC<Props> = (props) => {
   let dialogTitle: string = '';
   let dialogMessage: string = '';
 
-  if (examNotice && timedNotice) {
+  if (examNotice && timedNotice && !isNewSubmission) {
```

Comment:
workaround is to not show the dialog defining timed assessment if the time lapse between currentTime and the startTime is less than 10 seconds

Reply:
Why not show the dialog with the total time limit instead of not showing anything?  
Current logic means that users entering an exam will not get the examNotice, which defeats the point of the examNotice.

Reply:
actually anything will do for this one..
but even with the current logic, if the assessment is exam, user will still get the notification (you might see several lines after that particular one, `else if (examNotice)`)

but understand that from our discussion, we agreed on the showing of the time limit for new submission, so I'll do this one